docs/source/release_notes/release-notes.rst

```diff
@@ -80,7 +82,9 @@ develop
            (`Pull Request <https://github.com/palantir/atlasdb/pull/1727>`__)
 
     *    - |fixed| |improved|
-         - Cassandra depedencies have been bumped to newer versions; should fix a bug (#1654) that caused Atlas probing downed Cassandra nodes every few minutes to see if they were up and working yet to eventually take out the entire cluster by steadily building up leaked connections, due to a bug in the underlying driver.
+         - Cassandra depedencies have been bumped to newer versions; should fix a bug (`#1654 <https://github.com/palantir/atlasdb/issues/1654>`__) that caused
```

Comment:
Would be nice to have a stress-test for this, but wouldn't block the release on it.